### PR TITLE
update unattended_upgrades to 6.0.0

### PR DESCRIPTION
**Pull request by evgeni**
this brings Debian 11 support

6.0.0 technically drops support for Debian 9, and we still have one machine running this: discourse, but there are no changes that actually should break Debian 9 -- just removal from upstream CI

### CODE REVIEW

**Comment by evgeni** [comment]
hold on, we might just get a release of that thing out :D

**Comment by evgeni** [comment]
green, yay

**Comment by evgeni** [comment]
this escalated again a bit >.<

File: puppet/Puppetfile_forge [hunk 1]
```diff
@@ -16,9 +16,10 @@ mod 'puppet/jenkins', '2.0.0'
 mod 'puppet/letsencrypt', '5.0.0'
 mod 'puppet/nodejs', '8.0.0'
 mod 'puppet/selinux', '3.1.0'
-mod 'puppet/unattended_upgrades', '5.1.0'
+mod 'puppet/unattended_upgrades', '6.0.0'
 mod 'puppetlabs/apache', '5.4.0'
 mod 'puppetlabs/apt', '7.7.0'
+mod 'puppetlabs/augeas_core', '1.1.2'
```
Comment (ekohl) [inline puppet/Puppetfile_forge:22]: This should only be a testing dependency. All _core modules are bundled in the agent packages (but not the gem)

Comment (evgeni) [reply puppet/Puppetfile_forge:22]: but we also have this here:

https://github.com/theforeman/foreman-infra/blob/1c15a1d3fa616a316795b38b6a7dfeae3cde223e/puppet/Puppetfile_forge#L27

Comment (ekohl) [reply puppet/Puppetfile_forge:22]: That is probably an oversight.

Comment (evgeni) [reply puppet/Puppetfile_forge]: updated

**Review by ekohl — APPROVED** [review]

**Pull request merged**